install/kots/manifests/kots-preflight.yaml

```diff
@@ -95,3 +95,10 @@ spec:
               message: The cluster must contain at least 4 cores
           - pass:
               message: There are at least 2 cores in the cluster
+    - customResourceDefinition:
+        customResourceDefinitionName: certificate.cert-manager.io
```

Comment:
Consider adding `checkName: Cert-manager is installed` to give a friendly name to the check